chart/templates/networkpolicy.yaml

```diff
@@ -30,7 +30,7 @@ spec:
     - ports:
         - port: 443
         - port: 8443
-      to:
+    - to:
```

Comment:
This changes logic from an `and` to an `or`, why is this required? This basically allows communication with all pods on port 443 and 8443 as well as all ports on the control-plane

Reply:
Same as previous one. I misunderstand the grammar of `NetworkPolicyEgressRule`. Just revert back